news/2020-04-10-osu-spotlights-new-team-and-league-system.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "osu! Spotlights: New Team and League System"
+date: 2020-04-13 08:00:00 +0000
+---
+
+With our latest [Spotlight season] released very recently, we are underway reworking the osu! Spotlights from grounds up, starting with a brand new team and adding a whole new league system.
+
+![](image)
+
+Since March 2017 osu! Spotlights work in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea and give the curation and participation in the spotlights a whole new twist.
```

Comment:
```suggestion
Since March 2017, osu! Spotlights have worked in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea by giving curation and participation in the Spotlights a whole new twist.
```
word tense + better flow for the second sentence (reduced usage of `and`)